### DISCUSSION TITLE
Braid-HTTP ? *anyone*

### DISCUSSION THREAD

**Discussion by dubiouscript** [TiddlyWiki/TiddlyWiki5#8219] [General]
happened to read some reference to braid-http 
after scanning through some sections 
i wandered if this might have caught anyone else's attention ??? 

https://datatracker.ietf.org/doc/html/draft-toomim-httpbis-braid-http-04

https://github.com/braid-org/braid-spec

**Comment by Jermolene** [comment]
Hi @dubiouscript thank you I had not come across Braid-HTTP, and it is indeed very congruent with the sync engine in [MWS](https://github.com/Jermolene/TiddlyWiki5/pull/7915), I shall follow developments with interest. The "Subscribe" header is a nice way to progressively enhance a normal GET request with SSE semantics.
